akka-docs-dev/rst/scala/code/docs/http/scaladsl/server/FullTestKitExampleSpec.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2009-2014 Typesafe Inc. <http://www.typesafe.com>
+ */
+
+package docs.http.scaladsl.server
+
+// format: OFF
+
+//# source-quote
+import org.scalatest.{ Matchers, WordSpec }
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import akka.http.scaladsl.server._
+import Directives._
+
+class FullTestKitExampleSpec extends WordSpec with Matchers with ScalatestRouteTest {
+
+  val smallRoute =
+    get {
+      pathSingleSlash {
+        complete {
+          "Captain on the bridge!"
+        }
+      } ~
+      path("ping") {
+        complete("PONG!")
+      }
+    }
+
+  "The service" should {
+
+    "return a greeting for GET requests to the root path" in {
+      Get() ~> smallRoute ~> check {
+        responseAs[String] should contain("Captain on the bridge")
+      }
+    }
+
+    "return a 'PONG!' response for GET requests to /ping" in {
+      Get("/ping") ~> smallRoute ~> check {
+        responseAs[String] shouldEqual "PONG!"
+      }
+    }
+
+    "leave GET requests to other paths unhandled" in {
+      Get("/kermit") ~> smallRoute ~> check {
+        handled shouldBe false
+      }
+    }
+
+    "return a MethodNotAllowed error for PUT requests to the root path" in {
+      Put() ~> Route.seal(smallRoute) ~> check {
```

Comment:
just add a note or link somewhere in the docs what Route.seal is and why it is needed.